src/twisted/conch/test/test_keys.py

```diff
@@ -135,11 +135,28 @@ def test__guessStringType(self):
         self.assertEqual(
             keys.Key._guessStringType(keydata.publicECDSA_openssh), "public_openssh"
         )
+        self.assertEqual(
+            keys.Key._guessStringType(keydata.publicSKECDSA_openssh), "public_openssh"
+        )
+        self.assertRaises(
```

Comment:
I think that is best to have this in a separate test method... with a docstring explaining that OpenSSH certs are not supported.